src/jit/hwintrinsicxarch.cpp

```diff
@@ -677,17 +702,25 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
     // table-driven importer of simple intrinsics
     if (impIsTableDrivenHWIntrinsic(category, flags))
     {
-        if (!varTypeIsSIMD(retType) || ((flags & HW_Flag_BaseTypeFromArg) != 0))
+        if (category == HW_Category_MemoryStore ||
```

Comment:
NOTE: I had to move this block in my AVX PR (https://github.com/dotnet/coreclr/pull/16655) in order to deal with the intrinsics that have both generic and non-generic overloads.